README.md

```diff
@@ -99,3 +99,4 @@ This is useful if the mapping has changed in an incompatible way but the existin
 Depending on the size of the index, a large amount of memory may be consumed.
 
 There is one parameter which specifies which index to recreate: `transfers` or `aips`.
```

Comment:
This should be update because there's multiple parameters now and this one's required.